documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -36,12 +36,28 @@ ifdef::product[]
 For information about the Oracle Database versions that are compatible with this connector, see the link:{LinkDebeziumSupportedConfigurations}[{NameDebeziumSupportedConfigurations}].
 endif::product[]
 
-{prodname} ingests change events from Oracle by using the native LogMiner database package
+{prodname} ingests change events from Oracle by using the native LogMiner database package,
```

Comment:
Wasn't the `,` (comma) handling here correct in the old version? I would have expected for downstream it would have read "LogMiner database package or the ..." rather than "LogMiner database package, or the ...".  Am I mistaken here? :thinking:

Reply:
Sigh. Yeah, technically you're correct, and I was went back and forth about this, trying to balance moving the comma so that it gets inserted where it's not necessary, or having it render oddly with an extra leading space. It would seem that the original conditionalization would insert the comma immediately after the word `package`, but that's not how it renders in the built documentation. 
The other option would be offer two completely distinct sentences for the community and product versions.